http-poja/src/main/java/io/micronaut/http/poja/fork/netty/QueryStringDecoder.java

```diff
@@ -0,0 +1,411 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.micronaut.http.poja.fork.netty;
```

Comment:
rename this package to `util` or something and add javadoc that is is a fork on the netty class. If fork it we must also fork the tests.